libs/utils/utils.js

```diff
@@ -309,6 +309,7 @@ export const [setConfig, updateConfig, getConfig] = (() => {
         && (conf.useDotHtml ?? PAGE_URL.pathname.endsWith('.html'));
       config.entitlements = handleEntitlements;
       config.consumerEntitlements = conf.entitlements || [];
+      config.atvCaptionsKey = conf.atvCaptionsKey;
```

Comment:
isn't this set by the consumer? I don't think you need to set it here specifically